graph/client/src/app/ui-components/error-boundary.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEnvironmentConfig } from '@nx/graph/shared';
+import { useRouteError } from 'react-router-dom';
+
+export function ErrorBoundary() {
+  let error = useRouteError().toString();
+  console.error(error);
+  const environment = useEnvironmentConfig()?.environment;
+
+  let message = 'Failed to fetch data for graph. ';
+  if (environment === 'nx-console') {
+    message += 'Please refresh the page.';
+  } else {
+    message += 'Please rerun the command and refresh the page.';
```

Comment:
I would update the text slightly here. Maybe 'Please restart the grpah server and refresh the page'/
Because users might not know what 'the command' is/